apps/guide/src/content/01-home/02-whats-new.mdx

```diff
@@ -33,25 +31,9 @@ We have moved from VuePress to [Next.js](https://nextjs.org/)! The source can be
 
 ## Pages
 
-All content has been updated to use discord.js v14 syntax. The v13 version of the guide can be found at https://v13.discordjs.guide.
-
-### New
-
-- [Updating from v13 to v14](/additional-info/changes-in-v14.md): A list of the changes from discord.js v13 to v14
-- [Slash commands](/interactions/slash-commands.md): Registering, replying to slash commands and permissions
-- [Buttons](/interactions/buttons.md): Building, sending, and receiving buttons
-- [Select menus](/interactions/select-menus.md): Building, sending, and receiving select menus
-- [Threads](/popular-topics/threads.md): Creating and managing threads
-- [Builders](/popular-topics/builders.md): A collection of builders to use with your bot
-
-### Updated
-
-- Commando: Replaced with [Sapphire](https://sapphirejs.dev/docs/Guide/getting-started/getting-started-with-sapphire)
-- [Voice](/voice/): Rewritten to use the [_`@discordjs/voice`_](https://github.com/discordjs/discord.js/tree/main/packages/voice) package
-- [Command handling](/creating-your-bot/handling-command-interactions.md/): Updated to use slash commands
-  - Obsolete sections removed
-- _`client.on('message')`_ snippets updated to _`client.on(Events.InteractionCreate)`_
-  - [Message content became a privileged intent on August 31, 2022](https://support-dev.discord.com/hc/articles/4404772028055)
+- Pages have been revamped to account for our new [create-discord-bot](https://github.com/discordjs/discord.js/tree/main/packages/create-discord-bot) command-line interface.
+- Popular topic are now simply "topics" that detail usage of a particular concept of the API.
+- Focus is primarily on discocrd.js, so irrelevant topics have been removed. It may be better to visit the documentation of the package you are using to learn how to use them.
```

Comment:
```suggestion
- Focus is primarily on discord.js, so irrelevant topics have been removed. It may be better to visit the documentation of the package you are using to learn how to use them.
```